Dockerfile

```diff
@@ -0,0 +1,24 @@
+# we're using Golang 1.x (1.16.3 at the time of writing)
+#             and Debian buster
+FROM golang:alpine
```

Comment:
I guess this comment needs to be updated after shifting to `golang:alpine`

Reply:
Oh right, good catch.